test_runtime.sh

```diff
@@ -78,7 +78,7 @@ tar -xf  rootfs.tar.gz -C ${TESTDIR}
 cp runtimetest ${TESTDIR}
 
 pushd $TESTDIR > /dev/null
-ocitools generate --output config.json "${TEST_ARGS[@]}" --rootfs '.'
+ocitools generate --tty --output config.json "${TEST_ARGS[@]}" --rootfs '.'
```

Comment:
On Mon, Jul 18, 2016 at 03:41:25PM -0700, Mrunal Patel wrote:
> > -ocitools generate --output config.json "${TEST_ARGS[@]}" --rootfs '.'
> > +ocitools generate --tty --output config.json "${TEST_ARGS[@]}" --rootfs '.'
> 
> /dev/console is only setup if we set terminal to true. 

Can we stop adding “runC isn't compliant out of the box” workarounds
now that we're post 1.0.0-rc1?  If there's really a reason to not
setup /dev/console when terminal is false, can we PR the spec with
that information?